signalfx-org-metrics/metrics.yaml

```diff
@@ -1364,11 +1364,11 @@ sf.org.limit.customMetricTimeSeries:
   brief: Maximum number of custom MTS you can create in your organization within a 1 hour time window
   description: |
     Maximum number of custom MTS you can create in your organization in a 1 hour time window.
-    If you exceed this limit, Infrastructure Monitoring stops accepting datapoints for new custom MTS,
-    but keeps accepting datapoints for existing custom MTS. To see the overall number of custom metrics
-    you've defined, use the metric `sf.org.numCustomMetrics`.
```

Comment:
"defined" seems right,  why did you change it?

Reply:
It used to say "define", as in "To see the overall number of custom metrics you've **define**,..."

which is a sad and embarrassing grammar mistake.